File: Day1_Java/q15.java
import java.util.InputMismatchException;
import java.util.Scanner;

public class q15 {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the Number to Check whether it is Even or Odd");

        try {

            int n = sc.nextInt();
            if (n % 2 == 0) {

                System.out.println("Even: " + n);
            } else if (n % 2 == 1) {
                System.out.println("Odd: " + n);

            }

        } catch (InputMismatchException e) {
            System.out.println("Please Correct Data:");
        }
        sc.close();

    }
}

File: Questions_in_jS_Console_Based/q15.js
//Accept an integer and check whether it is an even number or odd.

let n = parseInt(prompt('Enter the Number to Check Even or ODD.'));
try {
    if (n % 2 == 0) alert(`Number is Even: ${n}`);
    else if (n % 2 == 1) alert('Odd', n);
    else alert("You have Entered something ")

} catch (error) {
    console.log(`Please Enter Correct Data`);

}

